exercise1/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using RestSharp;
+using Retrofit.Net;
+
+namespace exercise1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var dataCenter = "us17";
+            var apiKey = "ed37a130dbadee9671f725ebdf03583a-us17";
+            var url = $"https://{dataCenter}.api.mailchimp.com/3.0";
+            // init service
+            var listService = GetService<IListService>(url, apiKey);
+            // create a new list
+            var list = ObjectHelper.GetListObject();
+            Console.WriteLine(listService.CreateList(list).ResponseStatus);
+            // add an email to list
+            var listId = "ea602c2f5c";
```

Comment:
The listId should have the value from the response above

Reply:
done